frontend/server/src/test/resources/log4j2-json.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you add comments on each of the xml files on how you generated them?

Were there any existing xml files that now need to be deleted

Where are these xml files used?

Generally feels like we need to update internals guide with how metrics work